coverage/htmlfiles/coverage_html.js

```diff
@@ -595,10 +595,31 @@ coverage.wire_up_sticky_header = function () {
     updateHeader();
 };
 
+coverage.showContexts = function (e) {
+  span = e.target.nextElementSibling.nextElementSibling;
+  span_text = span.textContent;
+
+  if (/^[0-9,]+$/.test(span_text))
+  {
+    span.textContent = "";
+    span_text.split(",").forEach(function(s) {
+      ctx = contexts[s];
+      span.appendChild(document.createTextNode(ctx));
+      span.appendChild(document.createElement("br"));
```

Comment:
This creates slightly different HTML than the current code, so the styling needs to be adjusted to keep the right-alignment.